ipn/ipnstate/ipnstate.go

```diff
@@ -710,3 +711,25 @@ type DebugDERPRegionReport struct {
 	Warnings []string
 	Errors   []string
 }
+
+type ProgressStatus string
```

Comment:
What about calling this `UpdateStatus` ?  I see that we could conceivably use this type for other, non-update statuses, but I suspect we would want those to be their own type?

Reply:
calling it `SelfUpdateStatus` then - both "update" and "status" are so generic words that I was feeling the same issue you described with `UpdateStatus`